canonical-websites/design.ubuntu.com
# fixed use of cols in /brand section

## Done

* fixed up the use of cols in the /brand section

## QA

* download branch
* `./run`
* look at the pages in the [brand section](http://0.0.0.0:8011/brand/)
* see that they look ok and consistent

## Issue / Card

Fixes #78
